io.github.FedericoCalzoni.EventRecorder.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id": "io.github.FedericoCalzoni.EventRecorder",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "base": "com.riverbankcomputing.PyQt.BaseApp",
+    "base-version": "6.7",
+    "cleanup-commands": [
+        "/app/cleanup-BaseApp.sh"
+    ],
+    "command": "EventRecorder.py",
+    "modules": [
+        {
+            "name": "EventRecorder",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/FedericoCalzoni/EventRecorder/archive/refs/tags/v1.0.0.tar.gz",
+                    "dest": "EventRecorder",
+                    "sha256": "a324e76289859a8e26c4a5e9088880ea676578e1f773afffbb8314f001018b2a"
+                }
+            ],
+            "build-commands": [
+                "install -D EventRecorder/EventRecorder/EventRecorder.py /app/bin/EventRecorder.py",
+                "chmod +x /app/bin/EventRecorder.py",
+                "install -D EventRecorder/EventRecorder/logo.png /app/share/icons/hicolor/128x128/apps/io.github.FedericoCalzoni.EventRecorder.png",
+                "install -D EventRecorder/io.github.FedericoCalzoni.EventRecorder.desktop /app/share/applications/io.github.FedericoCalzoni.EventRecorder.desktop"
+            ]
+        }
+    ],
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--socket=wayland",
+        "--filesystem=home"
```

Comment:
why do you need a broad filesystem permission?

Reply:
I have reduced the filesystem permission to only include: 
"--filesystem=~/.config/EventRecorder.ini"

The app works, but the shown path of the opened file is not the one of the filesystem but rather:
/run/user/1000/doc/982531fc/prova1.csv

I don't know if there is a way to show the proper one (at least what the user expect to be).

Reply:
>  "--filesystem=~/.config/EventRecorder.ini"

Are you storing configs relative to HOME or XDG_CONFIG_HOME? The latter is the correct way and does not need any filesystem permission

Reply:
otherwise `--persist` is a better alternative.

Reply:
I made a new version supporting the XDG directories, and updated the manifest for the new version.